src/txn/txn_rollback_to_stable.c

```diff
@@ -1623,6 +1623,20 @@ __rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start
     }
 }
 
+/*
+ * __check_btree_modified --
+ *     Check that the rollback to stable btree is dirty or not.
```

Comment:
how about modifying the comment also as `btree is modified or not` instead of `dirty`?